draft-private-access-tokens.md

```diff
@@ -710,10 +792,9 @@ policy with the Issuer. In this policy, the Issuer does not enforce any limit on
 of tokens a given user will obtain.
 
 Origins request tokens from Clients and, upon successful redemption, the Origin knows
-the Client was able to request a token for the given (ANON_CLIENT_ID, ORIGIN_NAME) tuple. As a
+the Client was able to request a token for the given ORIGIN_NAME tuple. As a
 result, the Origin knows this is an authentic client.
 
-
```

Comment:
Might need to expand on the privacy/security implications in the considerations sections below on the mediator's choice of group size? It seems like group sizes of 1 or infinity (all the clients within an epoch) could have some slight concerns without the Issuer enforcing some size requirements?

Reply:
Yes, agreed.